pkg/provider/aws/resources/api_gateway.go

```diff
@@ -175,10 +175,14 @@ func (link *VpcLink) KlothoConstructRef() []core.AnnotationKey {
 
 // Id returns the id of the cloud resource
 func (res *VpcLink) Id() core.ResourceId {
+	name := "<no target>"
+	if res.Target != nil {
+		name = res.Target.Id().String()
+	}
```

Comment:
is this because of the vpclink v2? I dont think there will ever be a no target for v1 or did we see that